src/Ledger/Foreign/HSLedger.agda

```diff
@@ -88,6 +88,27 @@ HSScriptStructure : ScriptStructure ℕ ℕ ℕ
 HSScriptStructure = record { p1s = HSP1ScriptStructure ; ps = HSP2ScriptStructure }
 
 open import Ledger.Transaction
+open import Ledger.TokenAlgebra
+open import Data.Nat.Properties using (+-0-commutativeMonoid; _≟_)
+open import Data.Nat
+
+coinTokenAlgebra : TokenAlgebra
+coinTokenAlgebra = record
+         { PolicyId = ℕ
+         ; Value-Commutative-Monoid = +-0-commutativeMonoid
+         ; coin = id
+         ; inject = id
+         ; policies = λ x → ∅
+         ; size = λ x → 1 -- there is only ada in this token algebra
+         ; property = λ x → refl
+         ; coinIsMonoidMorphism = record { mn-homo = record
+                                                     { sm-homo = record { ⟦⟧-cong = λ z → z
+                                                       ; ∙-homo = λ x y → refl }
```

Comment:
Please align the `;`'s with the `{` above and change the indentation to two spaces